static/app/types/alerts.tsx

```diff
@@ -28,11 +29,15 @@ export type IssueAlertRuleActionTemplate = {
   prompt: string;
   enabled: boolean;
   actionType?: 'ticket' | 'sentryapp';
-  formFields?: {
-    [key: string]: IssueAlertRuleFormField;
-  };
+  formFields?:
+    | {
```

Comment:
@leeandher why does this type start with `|`?

Reply:
TSLint formats it like that by default if the two types can't fit on the same line when you use the `|` separator.